### PR TITLE
Document Trezor Methods

### PR DESCRIPTION
**Pull request by smk762**
closes: https://github.com/KomodoPlatform/developer-docs/issues/392
- [x] task::init_trezor::init
- [x] task::init_trezor::status
- [x] task::init_trezor::user_action
- [x] task::enable_utxo::init
- [x] task::enable_utxo::status
- [x] task::enable_utxo::user_action
- [x] task::enable_qtum::init
- [x] task::enable_qtum::status
- [x] task::enable_qtum::user_action
~~task::withdraw::init~~ _moved to other doc_
~~task::withdraw::status~~ _moved to other doc_
~~task::withdraw::cancel~~ _moved to other doc_
- [x] task::account_balance::init
- [x] task::account_balance::status
- [x] task::account_balance::cancel
- [x] can_get_new_address
- [x] get_new_address

### CODE REVIEW

**Review by sergeyboyko0791 — COMMENTED** [review]
A huge work! Thank you!
Just a few notes and suggestions

File: docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md
Comment (smk762) [reply docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md:730]: I've added a section at the bottom for this detail which is linked to where applicable in the other sections

File: docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md
Comment (smk762) [reply docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md:322]: done

File: docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md
Comment (smk762) [reply docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md:179]: done - just need to generate a couple of the errors to get the json

File: docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md
Comment (smk762) [reply docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md:830]: I think in general where request payloads are the same (or 80% same with the remaining optional) it is good to have a single method that can be used for both. It reduces confusion from user and overhead to create maintain docs. 

IMO the ideal solution would be a single "enable_coin" method which can be used on anything, with API + coins file able to determine which underlying method to use. Another enhancement would be allowing user to define [coins_config.json](https://github.com/KomodoPlatform/coins/blob/master/utils/coins_config.json) and derive the rpc_nodes / electrums etc from there - though I understand this would be a fairly big job and maybe better to plan towards for v3 of the API

**Comment by smk762** [comment]
I've made some updates to resolve review feedback - just waiting on @Canialon to provide an error response example for `FoundMultipleDevices` and it will be ready for re-review

**Comment by Canialon** [comment]
> I've made some updates to resolve review feedback - just waiting on @Canialon to provide an error response example for `FoundMultipleDevices` and it will be ready for re-review

```
{
  "mmrpc": "2.0",
  "result": {
    "status": "Error",
    "details": {
      "error": "Found multiple devices. Please unplug unused devices",
      "error_path": "init_hw.crypto_ctx.hw_client",
      "error_trace": "init_hw:151] crypto_ctx:248] crypto_ctx:354] hw_client:152] hw_client:126]",
      "error_type": "HwError",
      "error_data": "FoundMultipleDevices"
    }
  },
  "id": null
}
```

**Comment by Canialon** [comment]
Looks like it is no `trezor_integration` page there.. it wasn’t added to the sidebar 
image

**Comment by smk762** [comment]
fixed a little syntax that was causing error, should be good now :crossed_fingers:

**Comment by Canialon** [comment]
> fixed a little syntax that was causing error, should be good now 🤞

The sidebar works good now! thanks!!

**Comment by smk762** [comment]
There appears to be duplication of the withdraw task methods between this PR and https://github.com/KomodoPlatform/developer-docs/pull/395, which is ready for re-review. Once we merge https://github.com/KomodoPlatform/developer-docs/pull/395, I'll merge master into this branch and tweak the structure to use links instead of duplicated sections.

File: docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md
Comment (sergeyboyko0791) [reply docs/basic-docs/atomicdex-api-20-dev/trezor_integration.md:867]: I can't answer this suggestion correctly as I didn't work on `enable V2` feature, so I'll just reference to this comment https://github.com/KomodoPlatform/atomicDEX-API/issues/1006#issuecomment-960594323 and to your comment https://github.com/KomodoPlatform/atomicDEX-API/issues/1006#issuecomment-960663488 🙂

**Comment by smk762** [comment]
I've broken up this doc into sections, as some methods are not Trezor specific (e.g. coin activation and withdrawal tasks are possible without a Trezor). Sidebar now looks like below
image

**Review by Canialon — APPROVED** [review]

**Review by SirSevenG — APPROVED** [review]

**Pull request merged**
